packages/react-core/src/components/Pagination/PaginationOptionsMenu.tsx

```diff
@@ -128,9 +128,9 @@ export class PaginationOptionsMenu extends React.Component<PaginationOptionsMenu
         {title}
         <span className={css(paginationStyles.paginationMenuText)}>{` ${perPageSuffix}`}</span>
         {perPage === value && (
-          <i className={css(styles.optionsMenuMenuItemIcon)}>
+          <span className={css(styles.optionsMenuMenuItemIcon)}>
```

Comment:
Should now be a div [per core:](http://patternfly-v4.surge.sh/documentation/core/components/pagination)
```
<div class="pf-c-options-menu__menu-item-icon">
    <i class="fas fa-check" aria-hidden="true"></i>
</div>
```